plenum/server/node.py

```diff
@@ -1800,6 +1804,8 @@ def allLedgersCaughtUp(self):
                         .format(CATCH_UP_PREFIX, self),
                         extra={'cli': True})
             self.no_more_catchups_needed()
+            # select primaries after pool ledger caughtup
+            self.select_primaries()
```

Comment:
Should we check if `view_change_in_progress` is False?

Reply:
Or maybe we can check if the expected number of replicas is greater than the current one?